src/main/g8/scalafix/build.sbt

```diff
@@ -14,41 +18,93 @@ inThisBuild(
         url("https://example.com")
       )
     ),
-    addCompilerPlugin(scalafixSemanticdb),
-    scalacOptions ++= List(
-      "-Yrangepos",
-      "-P:semanticdb:synthetics:on"
-    )
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision
   )
 )
 
-publish / skip := true
+lazy val root = (project in file("."))
+  .aggregate(
+    rules.projectRefs ++
+      input.projectRefs ++
+      output.projectRefs ++
+      tests.projectRefs: _*
+  )
+  .settings(
+    publish / skip := true
+  )
```

Comment:
Is it needed ?

Reply:
without that, commands at the root like `scalafmtAll` won't aggregate, see https://github.com/sbt/sbt-projectmatrix#two-matrices

Reply:
I updated the name to have something more explicit than `root` (which should be a no-op compared to before)